src/main/java/com/dataguise/saas/controllers/MetadataController.java

```diff
@@ -0,0 +1,98 @@
+package com.dataguise.saas.controllers;
+
+import com.dataguise.saas.dto.ColumnViewUIFormatterDTO;
+import com.dataguise.saas.dto.SensitiveExpressionDTO;
+import com.dataguise.saas.dto.SourceTreeViewUIFormatterDTO;
+import com.dataguise.saas.exception.BadGatewayException;
+import com.dataguise.saas.exception.BadRequestException;
+import com.dataguise.saas.exception.InternalServerException;
+import com.dataguise.saas.service.ProtectionTaskService;
+import com.dataguise.saas.service.SourceHierarchyExplorerService;
+import com.dataguise.saas.service.TaskBuilderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.web.bind.annotation.*;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/controllers/MetadataController.java L15
    
    Using the &#39;.*&#39; form of import should be avoided - org.springframework.web.bind.annotation.*.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a294313783>*